types/react-native/index.d.ts

```diff
@@ -1107,6 +1107,43 @@ export interface TextInputIOSProps {
  * @see https://facebook.github.io/react-native/docs/textinput.html#props
  */
 export interface TextInputAndroidProps {
+    /**
+     * Determines which content to suggest on auto complete, e.g.`username`.
+     * To disable auto complete, use `off`.
+     *
+     * *Android Only*
+     *
+     * The following values work on Android only:
+     *
+     * - `username`
+     * - `password`
+     * - `email`
+     * - `name`
+     * - `tel`
+     * - `street-address`
+     * - `postal-code`
+     * - `cc-number`
+     * - `cc-csc`
+     * - `cc-exp`
+     * - `cc-exp-month`
+     * - `cc-exp-year`
+     * - `off`
+     */
```

Comment:
Is it really needed to list all of these in the comment? That will require keeping them up-to-date.

Reply:
The documentation is provided by https://github.com/facebook/react-native/blob/master/Libraries/Components/TextInput/TextInput.js#L438-L461 
I did not modify it, just copy & paste.